src/Castle.Windsor/MicroKernel/SubSystems/Conversion/ComponentModelConverter.cs

```diff
@@ -29,12 +29,6 @@ public class ComponentModelConverter : AbstractTypeConverter
 	{
 		public override bool CanHandleType(Type type)
 		{
-			// Mono 1.9+ thinks it can convert strings to interface
-			if (type.IsInterface)
```

Comment:
Hmmm, sounds like we are missing a test here ... let me give it another look tonight.